src/app/app.routing.ts

```diff
@@ -25,5 +25,14 @@ export const routes: Routes = [
   {
     path: 'terms',
     component: TermsComponent
+  },
+  {
+    path: 'initWithQuery',
```

Comment:
nit: A camel-case URL seems a bit unusual to me, maybe something like...

```
project/:project/dataset/:dataset/table/:table
```

Reply:
Done.